package.json

```diff
@@ -1,203 +1,83 @@
-    "semantic-release": "semantic-release pre && npm run build && cp -r .git dist && npm publish dist/ && semantic-release post",
-    "start": "webpack-dev-server --inline --progress --host 0.0.0.0 --port 8088",
-    "start-dev": "webpack-dashboard -t almighty -- webpack-dev-server --inline --progress --host 0.0.0.0 --port 8088",
-    "stats": "webpack --profile --json > stats.json",
-    "test": "npm-run-all --serial test:unit check-coverage test:func",
-    "test:func": "./run_functional_tests.sh",
-    "test:unit": "karma start",
-    "testserver-start": "webpack-dev-server --inline --progress --host 0.0.0.0 --port 8088",
-    "validate": "npm-run-all --parallel validate-webpack:* test:unit --serial check-coverage",
-    "validate-webpack:dev": "webpack-validator webpack.config.js --env.dev",
-    "validate-webpack:prod": "webpack-validator webpack.config.js --env.prod",
-    "webdriver-start": "./node_modules/protractor/bin/webdriver-manager update && ./node_modules/protractor/bin/webdriver-manager start",
-    "watch:test": "npm run test:unit -- --auto-watch --no-single-run",
-    "watch:library": "gulp watch"
-  },
-  "license": "Apache-2.0",
-  "contributors": [
-    {
-      "name": "Arunkumar S",
-      "email": "asrisail@redhat.com"
-    },
-    {
-      "name": "Jaivardhan Akumar",
-      "email": "jakumar@redhat.com"
-    },
-    {
-      "name": "Jyasveer Gotta",
-      "email": "jgotta@redhat.com"
-    }
-  ],
-  "keywords": [
-    "angular2",
-    "webpack",
-    "typescript",
-    "fabric8",
-    "fabric8-stack-analysis-ui"
-  ],
-  "homepage": "https://github.com/fabric8-ui/fabric8-recommender",
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/fabric8-ui/fabric8-recommender.git"
-  },
-  "bugs": {
-    "url": "https://github.com/fabric8-ui/fabric8-recommender/issues"
-  },
-  "engines": {
-    "node": ">= 8.3.0",
-    "npm": ">= 5.3.0"
+    "semantic-release": "semantic-release"
   },
+  "private": true,
   "dependencies": {
+    "@angular/animations": "^6.1.0",
+    "@angular/common": "^6.1.0",
+    "@angular/compiler": "^6.1.0",
+    "@angular/core": "^6.1.0",
+    "@angular/forms": "^6.1.0",
+    "@angular/http": "^6.1.0",
+    "@angular/platform-browser": "^6.1.0",
+    "@angular/platform-browser-dynamic": "^6.1.0",
+    "@angular/router": "^6.1.0",
     "c3": "^0.4.18",
-    "ngx-bootstrap": "^1.9.3",
-    "ngx-modal": "^0.0.29"
-  },
-  "peerDependencies": {
-    "@angular/common": "^4.4.6",
-    "@angular/core": "^4.4.6",
-    "@angular/forms": "^4.4.6",
-    "@angular/http": "^4.4.6",
-    "ngx-base": "^2.3.0",
-    "ngx-fabric8-wit": "^6.20.3",
-    "ngx-login-client": "^1.2.0",
-    "rxjs": "^5.5.2"
+    "core-js": "^2.5.4",
+    "ngx-base": "^3.1.1",
+    "ngx-bootstrap": "^3.0.1",
+    "patternfly": "^3.54.1",
+    "ngx-fabric8-wit": "^7.1.0",
+    "ngx-login-client": "^2.3.1",
+    "ngx-modal": "^0.0.29",
+    "rxjs": "^6.0.0",
+    "rxjs-compat": "^6.3.0",
```

Comment:
@invincibleJai I think this dependency should be removed as we now use rxjs6 for all dependencies (latest fabric8-wit) on both `package.json`